cypress/e2e/setup-scripts/seed_contributor.py

```diff
@@ -1,24 +1,25 @@
 """DB seed script for e2e cypress contributor tests."""
 
-import os.path
 import json
+import os.path
+
 from django.conf import settings
 from django.contrib.auth.models import User
-from projects.models import Project
 
+from projects.models import Project
 
 cypress_path = os.path.join(settings.BASE_DIR, "cypress/fixtures")
-print(f"Now loading the json users file from fixtures path: {cypress_path}") # /app/cypress/fixtures
+print(f"Now loading the json users file from fixtures path: {cypress_path}")  # /app/cypress/fixtures
 
-with open( os.path.join(cypress_path, "users.json"), 'r') as f:
+with open(os.path.join(cypress_path, "users.json"), 'r') as f:
```

Comment:
A suggestion for later :)
I personally think that pathlib is better to use

```suggestion
from pathlib import Path

cypress_path = Path(settings.BASE_DIR, "cypress/fixtures")
print(f"Now loading the json users file from fixtures path: {str(cypress_path)}")  # /app/cypress/fixtures

with (cypress_path / "users.json").open('r') as f:
```

Reply:
By the way, why black didn't fix quotes? `.open('r')`? 

Is it related to `--check` argument?
![image](https://github.com/ScilifelabDataCentre/stackn/assets/8545082/f3bfb43e-b269-45bd-9d90-f6df2dbf38f8)

Reply:
Yes, because check makes it only report problems, not auto-fixing them. I decided that black makes so many style changes that I thought it wise to first run it with check for a while and then to discuss it with the entire team when everyone is back.

Reply:
Oh, I see

May be we should add a comment about it here, so that it could be brought up later?

Reply:
I can add a Jira issue to review the pre-commit setup and complement with additional future checks.

Reply:
I would personally write it here, as it would immediately answer the question, why was it done that way. Or you could make a jira task and add a link to it here.

But in any case, some explanation should be left here, because it's a temporary solution

Reply:
A Jira task to revisit this: https://scilifelab.atlassian.net/browse/SS-490

Reply:
As a sidenote, what do you think about pathlib.
https://github.com/ScilifelabDataCentre/stackn/pull/39/files#r1246640287

It's not a big deal, just curious